storage/postgres/tree_storage.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package postgres
+
+import (
+	"database/sql"
+
+	"github.com/golang/glog"
+)
+
+// OpenDB opens a database connection for all MySQL-based storage implementations.
+func OpenDB(dbURL string) (*sql.DB, error) {
```

Comment:
is dbURL a URL still, or should it be something about db connection parameters (or whatever terminology PGSQL uses for this)